Dockerfile

```
@@ -6,7 +6,8 @@
 ARG BASE_YUM_REPO=testing
 ARG BASE_OSG_SERIES=3.5
 
-FROM opensciencegrid/software-base:$BASE_OSG_SERIES-el7-$BASE_YUM_REPO AS xcache
+#FROM opensciencegrid/software-base:$BASE_OSG_SERIES-el7-$BASE_YUM_REPO AS xcache
+FROM opensciencegrid/software-base:$BASE_OSG_SERIES-el7-bh AS xcache
```

```suggestion
FROM opensciencegrid/software-base:$BASE_OSG_SERIES-el7-$BASE_YUM_REPO AS xcache
```
looks like a leftover from testing?

---

never mind, just saw your line about go-crond.